Add example web-summarizer demo

gokturkberke opened the pull request:
This PR adds a simple example demonstrating:
- Fetching text from a static web page (requests + BeautifulSoup)
- Hierarchical summarization using OpenAI
- Short summary + bullet points

ed-donner commented:
Hey - thanks so much for sharing. This PR seems to include files outside the community contributions folders. Let me know if this was intentional - otherwise please move all changes within community contributions when you get a chance. Thanks so much
Ed

gokturkberke commented:
Hi Ed, thanks a lot for pointing that out  You’re absolutely right — I mistakenly added files outside the community contributions folder. I’ve just fixed it and moved everything into the correct place. Really appreciate that and sorry for the mix-up!

ed-donner commented:
Thanks for making this valuable contribution and sharing your work with us all.
Ed
